var/bazel/rules/custom_proto_rules.bzl

```diff
@@ -0,0 +1,59 @@
+load("@bazel_skylib//lib:paths.bzl", "paths")
+load("@grpc//bazel:python_rules.bzl", "py_proto_library")
+
+def _proto_package_impl(ctx):
+    package_name = ctx.label.name
+    package_name = ctx.attr.package_name
+    all_outputs = []
+    init_file = ctx.actions.declare_file(paths.join(package_name,'__init__.py'))
+    ctx.actions.write(init_file, "")
+    all_outputs.append(init_file)
+    for d in ctx.attr.deps:
+        for f in d.files.to_list():
+            out = ctx.actions.declare_file(paths.join(package_name, f.basename))
+            all_outputs.append(out)
+            ctx.actions.run_shell(
+                outputs=[out],
+                inputs=depset([f]),
+                arguments=[f.path, out.path, paths.dirname(out.path)],
+                command="mkdir -p $3 && cp $1 $2"
+            )
+    return [DefaultInfo(files = depset(all_outputs), runfiles = ctx.runfiles(all_outputs)),
+            PyInfo(transitive_sources = depset(), imports = depset(direct = [paths.dirname(ctx.build_file_path)]))
+            ]
+
+proto_package = rule(
+    implementation = _proto_package_impl,
+    attrs = {
+        "package_name": attr.string(mandatory=True),
+        "deps": attr.label_list(allow_files=True),
+    },
+)
+
+# macro to create a regular python package with protobuf files
```

Comment:
there is not a lot of intelligence in this macro; right now it will generate the protobufs and then copy all the files in the same folder (the output package).